daemon/algod/api/algod.oas2.json

```diff
@@ -3639,6 +3643,35 @@
           "x-algorand-format": "uint64"
         }
       }
+    },
+    "SimulationLogLimits": {
+      "description": "The set of limits on log opcodes during simulation.",
+      "type": "object",
+      "required": [
+        "max-log-calls",
+        "max-log-size"
+      ],
+      "properties": {
+        "max-log-calls": {
+          "description": "The maximum log calls one can make during simulation",
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        },
+        "max-log-size": {
+          "description": "The maximum byte number to log during simulation",
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        }
+      }
+    },
+    "SimulationEvalChanges": {
+      "description": "The set of parameters and limits override during simulation.",
+      "type": "object",
+      "properties": {
+        "log-limits": {
+          "$ref": "#/definitions/SimulationLogLimits"
```

Comment:
Sorry to backtrack, but I think I saw this PR in an intermediate state before where `max-log-calls` and `max-log-size` where direct properties in `SimulationEvalChanges`, and I think I like that better. I.e. remove `SimulationLogLimits` and put its properties inside of `SimulationEvalChanges`. What do you think of that?

This way we don't have to define a new object for every type of eval change in the future, and I like that there's a potential symmetry between `logic.EvalConstants` and `SimulationEvalChanges`.

Reply:
I am fine with flattening the structure, i.e., make `max-log-*` fields directly under `eval-changes`.

Actually this somewhat makes maintaining the structures of `SimulationEvalChanges` easier and casting them from `Simulate.ResultEvalConstants` much easier.

Reply:
see [flatten eval-changes resp fields](https://github.com/algorand/go-algorand/pull/5247/commits/4a12bdc963e70f6d436a1f8534e78e603a1fe494)